source/connect.txt

```diff
@@ -17,8 +28,199 @@ Connect to MongoDB
    /connect/stable-api
    /connect/csot
 
-- :ref:`pymongo-tls`
-- :ref:`pymongo-network-compression`
-- :ref:`pymongo-server-selection`
-- :ref:`pymongo-stable-api`
-- :ref:`pymongo-csot`
\ No newline at end of file
+Overview
+--------
+
+This page contains code examples that show how to connect your Python application
+to MongoDB with various settings.
+
+.. tip::
+
+  To learn more about the connection options on this page, see the link
+  provided in each section.
+
+To use a connection example from this page, copy the code example into the
+:ref:`sample application <pymongo-connect-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as ``<hostname>``, with
+the relevant values for your MongoDB deployment.
+
+.. _pymongo-connect-sample:
+
+.. include:: /includes/usage-examples/sample-app-intro.rst
+
+.. literalinclude:: /includes/usage-examples/connect-sample-app.py
```

Comment:
Can this use the same sample app as authentication?

Reply:
sure--i just made the auth sample a little more generic to accommodate